src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -1282,6 +1282,19 @@ public void ShowUpdatePackages(ShowUpdatePackageOptions updatePackageOptions)
             }
         }
 
+        /// <summary>
+        /// Used for package namespace Apex tests which require All option in package sources.
+        /// </summary>
+        internal void SetPackageSourceOptionToAll()
```

Comment:
You can probably keep this async to avoid the JTF run. 

You can also already access both SelectedSource and PackageSources, so changing it should be easy without a method here.

Reply:
I removed change from here and moved to Apex contract.

Reply:
Now no change in product except `InternalsVisibleTo`.